File: 백준/Silver/1427. 소트인사이드/소트인사이드.cc
#include <iostream>
#include <algorithm>

using namespace std;

bool compare(int a, int b) {
	return a > b;
}
int main() {
	string num;
	cin >> num;
	
	sort(num.begin(), num.end(), compare);
	
	cout << num << endl;
	
	return 0;
}

File: 백준/Silver/1427. 소트인사이드/소트인사이드.cs
using System;

namespace Algorithm_Practice
{
    class Program
    {
        static void Main(string[] args) 
        {
            string num = Console.ReadLine();
            int[] arr = new int[num.Length];

            for (int i = 0; i < num.Length; i++) {
                arr[i] = int.Parse(num.Substring(i, 1));
            }

            Array.Sort(arr);
            Array.Reverse(arr);

            string answer = "";
            for (int i = 0; i < arr.Length; i++) {
                answer += arr[i].ToString();
            }

            Console.WriteLine(answer);
        }
    }
}

